webui/firebaseConfig.sample.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
-import { setDefaultTimeout } from 'cucumber';
-
-setDefaultTimeout(15000);
```

Comment:
nit: seems like git got confused about what happened here -- is there a way to make it reflect that this is actually `webui/firebaseConfig.js.sample -> webui/firebaseConfig.sample.ts`?

Reply:
whether it's a move or create is up to git to decide. The content also change quite a lot here too so I don't think it's doable.

Reply:
Looks like [`git mv`](https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/pull/70#discussion_r432076489) could fix it

Reply:
like, add the old file back, delete the new file, commit that, then do `git mv` again? would that survive squashing? don't really feel like doing that to mess with git honestly.

Reply:
I can probably give it a try, then. It will be cleaner and will also make it much easier for me to review the change.

Reply:
Tried
1. add back old file, remove new file, commit
2. move old file to new file, commit
3. update new file to the updated content, commit
The PR still showing add + remove instead of rename + updated. I pushed that to `cypress-git-mv` branch. See:
https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/compare/cypress-git-mv?expand=1

Reply:
Gah, what a pain. Thank you for trying. Resolving this discussion.